<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>951</width>
         <height>731</height>
        </rect>
       </property>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>951</width>
          <height>711</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>932</width>
           <height>1218</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="category_frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLineEdit" name="category_poisk">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>891</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Поиск...</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>960</x>
         <y>0</y>
         <width>341</width>
         <height>721</height>
        </rect>
       </property>
       <widget class="QPushButton" name="itog_cena">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>650</y>
          <width>341</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1144</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>340</width>
          <height>651</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>321</width>
           <height>1218</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="itog_frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
